contrib/tracking.go

```diff
@@ -0,0 +1,256 @@
+package contrib
+
+/*
+#include "tracking.h"
+*/
+import "C"
+import (
+	"image"
+
+	"gocv.io/x/gocv"
+)
+
+// Tracker - the base interface for object tracking
+//
+// s.a: https://docs.opencv.org/master/d0/d0a/classcv_1_1Tracker.html
+type Tracker interface {
+	// Init initializes the tracker with a known bounding box that surrounded the target
+	// note: this can only be called once. if you loose the object, you have to Close() the instance,
+	// create a new one, and call Init on it again
+	//
+	// https://docs.opencv.org/master/d0/d0a/classcv_1_1Tracker.html#a4d285747589b1bdd16d2e4f00c3255dc
+	Init(image gocv.Mat, boundingBox image.Rectangle) bool
+
+	// Update the tracker, return new bounding box and a bool, if it lost it.
```

Comment:
> Update updates the tracker, returns a new bounding box and a boolean determining whether the tracker lost the target.

(correct me if this is wrong but that's what I understand)

Reply:
yea, better.